### PR TITLE
changes that were required in VacDoc to resolve issue of wrong doses along with invoice update

### PR DESCRIPTION
**Pull request by basitshah94**
…sent in messages

### CODE REVIEW

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts [hunk 1]
```diff
@@ -72,7 +76,12 @@ export class VaccineAlertPage implements OnInit {
     await this.alertService.getChild(this.numOfDays, this.clinicId).subscribe(
       res => {
         if (res.IsSuccess) {
+          this.Childs = "";
```
Comment (thezeeshanasghar) [inline src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:79]: @basitshah94 
This is not needed.

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts [hunk 2]
```diff
@@ -72,7 +76,12 @@ export class VaccineAlertPage implements OnInit {
...
           this.Childs = res.ResponseData;
+
+          console.log("Childs Saved in this.Childs");
+          console.log(res.ResponseData);
```
Comment (thezeeshanasghar) [inline src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:83]: @basitshah94 
remvoe these two console log

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts [hunk 3]
```diff
@@ -140,27 +149,34 @@ export class VaccineAlertPage implements OnInit {
...
     });
     await loading.present();
 
     for (let i = 0; i < child.length; i++) {
+      console.log("");
+      console.log("message No = " + (i + 1));
       let message = this.generateSMS(child[i]);
-      await this.sendAlertMsg(child[i].ChildId, child[i].Child.User.MobileNumber, message);
+
+      // await this.sendAlertMsg(child[i].ChildId, child[i].Child.User.MobileNumber, message);
```
Comment (thezeeshanasghar) [inline src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:164]: @basitshah94 
please remove your debug console logs and commit again

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts [hunk 4]
```diff
@@ -140,27 +149,34 @@ export class VaccineAlertPage implements OnInit {
...
   generateSMS(schedule) {
     var sms1 = 'Reminder: Vaccination for ';
-
     sms1 += schedule.Child.Name + ' is due on ' + schedule.Date;
-
     sms1 += ' (' + schedule.Dose.Name + ' )';
 
+    console.log("message => Id = " + schedule.Id);
+    console.log(sms1);
+
```
Comment (thezeeshanasghar) [inline src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:176]: @basitshah94 
same here

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts (hunk 2 above)
Comment (basitshah94) [reply src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:83]: ok

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts (hunk 3 above)
Comment (basitshah94) [reply src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:163]: done and pushed

File: src/app/members/alert/vaccine-alert/vaccine-alert.page.ts (hunk 1 above)
Comment (basitshah94) [reply src/app/members/alert/vaccine-alert/vaccine-alert.page.ts:79]: done and pushed

**Pull request merged**